src/Ext/Bullet/Trajectories/PhobosTrajectory.cpp

```diff
@@ -62,6 +62,10 @@ void TrajectoryTypePointer::LoadFromINI(CCINIClass* pINI, const char* pSection)
 	if (_ptr)
 	{
 		_ptr->Trajectory_Speed.Read(exINI, pSection, "Trajectory.Speed");
+
+		if (abs(_ptr->Trajectory_Speed) < 1e-10)
+			_ptr->Trajectory_Speed = 0.001;
```

Comment:
```suggestion
		_ptr->Trajectory_Speed = Math::max(0.001,_ptr->Trajectory_Speed);
```
btw can Trajectory.Speed be negative?

Reply:
I am not sure if anyone is using this "feature", so I just make sure it is not 0. Let me get rid of it.